### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by shanberg1**

### CODE REVIEW

File: ARM-wvd-templates/HCI/Readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Azure Virtual Desktop with Stack HCI
+
+Use the Full Configuration ARM templates if you are looking for more configurations in your Azure Virtual Desktop session hosts deployed on Azure Stack HCI.
+
+## Deploy  Azure Virtual Desktop on Azure Stack HCI
+This ARM template will setup Azure Virtual Desktop on Azure Stack HCI by creating a new host pool and workspace, creating the session hosts on the HCI cluster, joining the domain to downloading and installing the Azure Virtual Desktop agents and register them to the host pool.
+Check out the template [here](https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/HCI/QuickDeploy/CreateHciHostpoolQuickDeployTemplate.json)
+<a  href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fgithub.com%2FAzure%2FRDS-Templates%2Fblob%2Fmaster%2FARM-wvd-templates%2FHCI%2FQuickDeploy%2FCreateHciHostpoolQuickDeployTemplate.json"  target="_blank">
```
Comment (kkambow) [inline ARM-wvd-templates/HCI/Readme.md:8]: Can this deploy button come in the next link, alongside with Visualize button?
This link is not redirecting, I believe we need to give the raw github link here.

File: ARM-wvd-templates/HCI/Readme.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+<a  href="http://armviz.io/#/?load=https%3A%2F%2Fgithub.com%2FAzure%2FRDS-Templates%2Fblob%2Fmaster%2FARM-wvd-templates%2FHCI%2FQuickDeploy%2FCreateHciHostpoolQuickDeployTemplate.json"  target="_blank">
+<img  src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+## Add new Azure Virtual Desktop session hosts to an existing host pool
+This ARM template will add new Azure Virtual Desktop machines to your existing host pool.
+Check out the template [here](https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/HCI/QuickDeploy/AddHciVirtualMachinesQuickDeployTemplate.json)
+
+<a  href="https://portal.azure.com/#create/Microsoft.Template/uri/https://github.com/Azure/RDS-Templates/blob/master/ARM-wvd-templates/HCI/QuickDeploy/AddHciVirtualMachinesQuickDeployTemplate.json"  target="_blank">
```
Comment (kkambow) [inline ARM-wvd-templates/HCI/Readme.md:20]: This link is not redirecting correctly, I believe we need to give the raw GitHub link here.

File: ARM-wvd-templates/HCI/Readme.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Azure Virtual Desktop with Stack HCI
+
+Use the Full Configuration ARM templates if you are looking for more configurations in your Azure Virtual Desktop session hosts deployed on Azure Stack HCI.
+
+## Deploy  Azure Virtual Desktop on Azure Stack HCI
+This ARM template will setup Azure Virtual Desktop on Azure Stack HCI by creating a new host pool and workspace, creating the session hosts on the HCI cluster, joining the domain to downloading and installing the Azure Virtual Desktop agents and register them to the host pool.
+Check out the template [here](https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/HCI/QuickDeploy/CreateHciHostpoolQuickDeployTemplate.json)
+<a  href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FHCI%2FQuickDeploy%2FCreateHciHostpoolQuickDeployTemplate.json"  target="_blank">
```
Comment (kkambow) [inline ARM-wvd-templates/HCI/Readme.md:8]: Move the deploy to next line?

File: ARM-wvd-templates/HCI/Readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Azure Virtual Desktop with Stack HCI
+
+Use the Full Configuration ARM templates if you are looking for more configurations in your Azure Virtual Desktop session hosts deployed on Azure Stack HCI.
+
+## Deploy  Azure Virtual Desktop on Azure Stack HCI
+This ARM template will setup Azure Virtual Desktop on Azure Stack HCI by creating a new host pool and workspace, creating the session hosts on the HCI cluster, joining the domain to downloading and installing the Azure Virtual Desktop agents and register them to the host pool.
+Check out the template [here](https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/HCI/QuickDeploy/CreateHciHostpoolQuickDeployTemplate.json)
+
+<a  href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FHCI%2FQuickDeploy%2FCreateHciHostpoolQuickDeployTemplate.json"  target="_blank">
```
Comment (kkambow) [inline ARM-wvd-templates/HCI/Readme.md:9]: The link should be for long-form template, not the quick deploy one.

File: ARM-wvd-templates/HCI/Readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+<a  href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FHCI%2FQuickDeploy%2FCreateHciHostpoolQuickDeployTemplate.json"  target="_blank">
+<img  src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+## Add new Azure Virtual Desktop session hosts to an existing host pool
+This ARM template will add new Azure Virtual Desktop machines to your existing host pool.
+Check out the template [here](https://raw.githubusercontent.com/Azure/RDS-Templates/master/ARM-wvd-templates/HCI/QuickDeploy/AddHciVirtualMachinesQuickDeployTemplate.json)
+
+<a  href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FHCI%2FQuickDeploy%2FAddHciVirtualMachinesQuickDeployTemplate.json"  target="_blank">
```
Comment (kkambow) [inline ARM-wvd-templates/HCI/Readme.md:21]: The link should be for long-form template, not the quick deploy one.

**Review by kkambow — CHANGES_REQUESTED** [review]

**Review by kkambow — APPROVED** [review]

**Review by walrusmcd — APPROVED** [review]
looks great

**Pull request merged**